core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -41,6 +41,15 @@ oppia.directive('multipleChoiceEditor', [function() {
         // Note that a questionId generated in this way may contain spaces,
         // since it is just the state name.
         $scope.questionId = $scope.getUniqueId();
+
+        $scope.getHash = function(suffix) {
```

Comment:
OK makes sense. Could you add a comment to this function with this explanation? Thanks!